src/components/ProductDetail/ProductDetail.js

```diff
@@ -140,6 +144,23 @@ class ProductDetail extends Component {
           quantity
         }
       ]);
+
+      // If no errors occurred, track action
+      if (data) {
+        // The response data will be in either `createCart` or `addCartItems` prop
+        // depending on the type of user, either authenticated or anonymous.
+        const { cart } = data.createCart || data.addCartItems;
+
+        this.trackAction({
+          variant: {
+            ...selectedVariant,
+            cart_id: cart._id, // eslint-disable-line camelcase
+            quantity
+          },
+          optionId: selectedOption ? selectedOption._id : null,
+          action: "Product Added"
```

Comment:
We should make `Product Added` a constant and export from either `lib/tracking/constants.js` or from `lib/tracking/trackProduct.js`.